lib/Connection.js

```diff
@@ -249,7 +249,8 @@ class Connection {
   // We want to hide `auth`, `agent` and `ssl` since they made
   // the logs very hard to read. The user can still
   // access them with `instance.agent` and `instance.ssl`.
-  [inspect.custom](depth, options) {
+  [inspect.custom]() {
+    // eslint-disable-next-line no-unused-vars
     const { authorization, ...headers } = this.headers;
```

Comment:
Note to reviewer: Even though `authorization` is not being used, its declaration is still necessary to remove `authorization` from `headers`